pkgs/top-level/python-packages.nix

```diff
@@ -12171,7 +12171,6 @@ in {
 
   google_apputils = buildPythonPackage rec {
     name = "google-apputils-0.4.1";
-    disabled = isPy3k;
```

Comment:
Does the duplicated removal generate a conflict and I need to remove it or git handles it?